index.html

```diff
@@ -101,7 +96,252 @@ <h1>I'm Adeyemi Glad to see you!</h1>
+                  </li>
+                </div>
+                <div class="textTagcss_3">
+                  <li>
+                    <div class="Css_3">
+                      <p>css</p>
+                    </div>
+                  </li>
+                </div>
+                <div class="textTagjava_3">
+                  <li>
+                    <div class="java_3">
+                      <p>javascript</p>
+                      </div>
+                  </li>
+                </div>
+              </ul>
+              <div id="Action_bttn_3">
+                <div class="projbttn_3">
+                  <div class="click_bttn_3">
+                  <button href="#SeeProject" >See Project</button>
+                </div>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="cardWork_4">
+            <img id="snap_portfolio_4"/>
+            <div class="leftBlock_4">
+              <div class="pryTxt_head_4">
+                <div class="text_head_4">
+                  Multi-Post
+                  Stories
+                </div>
+                <div class="frame_card_4">
+                  <div class="Client_card_4">
+                    <div class="Client_txt_4">
+                      CANOPY
+                    </div>
+                  </div>
+                  <span class="Counter_4"></span>
+                  <div class="Role_card_4">
+                    <div class="Role_txt_4">
+                      Back End Dev
+                    </div>
+                  </div>
+                  <span class="Counter2_4"></span>
+                  <div class="Year_card_4">
+                    <p class="Year_txt_4">
+                      2015
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div class="pryText_body_4">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+              </div>
+              <ul class="Tags_4">
+                <div class="textTag_Hml_4">
+                  <li >
+                    <div class="Hml_4">
+                      <p>html</p>
+                      </div>
+                  </li>
+                </div>
+                <div class="textTagcss_4">
+                  <li>
+                    <div class="Css_4">
+                      <p>css</p>
+                    </div>
+                  </li>
+                </div>
+                <div class="textTagjava_4">
+                  <li>
+                    <div class="java_4">
+                      <p>javascript</p>
+                      </div>
+                  </li>
+                </div>
+              </ul>
+              <div id="Action_bttn_4">
+                <div class="projbtn_4">
+                  <div class="click_bttn_4">
+                  <button href="#SeeProject" >See Project</a>
+                </div>
```

Comment:
Kindly remove the **href=#seeProject**. href works with the `a` tag and not a button. A `button` tag uses a type of submit.
 
Also, kindly style your button in line with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) template.

| Your Design |Figma Design |
| :---:   | :---: | 
|![wrongstyle](https://user-images.githubusercontent.com/73854999/200114623-bc3ed779-bee7-4530-a66c-2b72ec17f41d.PNG)| ![mic-btn-style](https://user-images.githubusercontent.com/73854999/200114270-39255bd8-c016-4da2-bf98-178f5f5a7e56.PNG)|